policyengine_us/variables/gov/states/va/tax/income/spouse_tax_adjustment/va_personal_exemption_head.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class va_personal_exemption_head(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Virginia aged/blind exemption"
+    defined_for = StateCode.VA
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2022-760-instructions.pdf#page=19"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.va.tax.income.spouse_head_adjustment
+        aged_blind_count = tax_unit("blind_head", period) + (
```

Comment:
blind_head = tax_unit("blind_head", period)
blind_spouse = tax_unit("blind_spouse", period)
total_blind = blind_head + blind_spouse

Reply:
same for age